File: week1/Emopusta (C# && C++)/C#/TwoSum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for(int i = 0; i<nums.Length;i++){
            for(int j = i+1; j<nums.Length;j++){
                if (nums[i]+nums[j] == target) return [i,j];
            }
        }
        return [0,0];
    }
}

File: week1/TugceErdemliAlSolWeek1/TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """

        num_indices = {}
        for i, num in enumerate(nums):
            complement = target - num
            if complement in num_indices:
                return ([num_indices[complement], i])
            num_indices[num] = i
